app/sdil/config/FilterWiring.scala

```diff
@@ -44,7 +46,9 @@ trait FilterWiring extends CommonWiring {
   //TODO change to DefaultSessionCookieBaker when 2.6 available platform wide
```

Comment:
Is this TODO still relevant, if so then why can't it be done?

Reply:
code changed to DefaultSessionCookieBaker